lib/ansible/vars/plugins.py

```diff
@@ -106,16 +79,23 @@ def _plugin_should_run(plugin, stage):
 
 
 def get_vars_from_path(loader, path, entities, stage):
-
     data = {}
+    if vars_loader._paths is None:
+        # cache has been reset, reload all()
+        _prime_vars_loader()
```

Comment:
I used vars_loader._paths here as opposed (for example) vars_loader._search_paths because I'm assuming we don't want to extend `OLD_PLUGIN_CACHE_CLEARING` to new caches.